src/components/views/PlayWhites.js

```diff
@@ -4,52 +4,73 @@ import BaseContainer from "components/ui/BaseContainer";
 import "styles/views/WhiteCardSelection.scss";
 import Header from "./Header";
 import CardButton from "../ui/CardButton";
-import MenuContainer from "../ui/MenuContainer";
+import {api, handleError} from "../../helpers/api";
+
+//TODO: fetch a game of a random user (todo) and your white cards (done)
 
 
 const PlayWhites = () => {
   // use react-router-dom's hook to access the history
   const history = useHistory();
+  const [cards, setCards] = useState(null)
+  const userId = localStorage.getItem("id")
+  const token = localStorage.getItem("token")
 
   // define a state variable (using the state hook).
   // if this variable changes, the component will re-render, but the variable will
   // keep its value throughout render cycles.
   // a component can have as many state variables as you like.
   // more information can be found under https://reactjs.org/docs/hooks-state.html
 
+    // fetch the blackCards from the server (it is the server's responsibility to give us 8 cards)
```

Comment:
Black cards in comment, white cards in function?

Reply:
done (in other branch)